ipn/ipnlocal/local.go

```diff
@@ -4140,12 +4142,33 @@ func (b *LocalBackend) setNetMapLocked(nm *netmap.NetworkMap) {
 	}
 	// Third pass, actually delete the unwanted items.
 	for k, v := range b.nodeByAddr {
-		if !v.Valid() {
+		if v == 0 {
 			delete(b.nodeByAddr, k)
 		}
 	}
 }
 
+func (b *LocalBackend) updatePeersFromNetmapLocked(nm *netmap.NetworkMap) {
+	if nm == nil {
+		b.peers = nil
+		return
+	}
+	// First pass, mark everything unwanted.
+	for k := range b.peers {
+		b.peers[k] = tailcfg.NodeView{}
```

Comment:
maybe have an explicit var

```
var invalidNode tailcfg.NodeView
for k := range b.peers {
  b.peers[k] = invalidNode
}
```

Reply:
The current way is consistent with what we do elsewhere. We can change it to `zero` if/when that happens.